### PR TITLE
Conditionally build lastlog funcionality and documentation

### PR DESCRIPTION
**Pull request by ikerexxe**
Partially resolves: https://github.com/shadow-maint/shadow/issues/674

### CODE REVIEW

File: configure.ac [hunk 1]
```diff
@@ -746,6 +758,7 @@ echo "	yescrypt passwords encryption:	$with_yescrypt"
 echo "	nscd support:			$with_nscd"
 echo "	sssd support:			$with_sssd"
 echo "	subordinate IDs support:	$enable_subids"
+echo "	enable lastlog:		$enable_lastlog"
```
Comment (hallyn) [inline configure.ac:761]: The spacing here doesn't look right (even when I wget the patch and remove the first lines).  I think you need to remove the space after the :, and add two tabs.

**Review by hallyn — CHANGES_REQUESTED** [review]
Thanks - looks good to me except for one cosmetic comment.

File: configure.ac (hunk 1 above)
Comment (ikerexxe) [reply configure.ac]: There were 2 tabs, and I think another one was missing. Please check it on your setup and if it works close this conversation.

Comment (jubalh) [reply configure.ac]: I checked out now and spacing is correct now.

**Pull request merged**